Source/WTF/wtf/TZoneMallocInlines.h

```diff
@@ -36,15 +36,21 @@ WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
 
 #define WTF_MAKE_TZONE_ALLOCATED_INLINE(typeName) WTF_MAKE_FAST_ALLOCATED
 #define WTF_MAKE_TZONE_ALLOCATED_IMPL(typeName) struct WTFTzoneMallocSemicolonifier##typeName { }
-#define WTF_MAKE_TZONE_ALLOCATED_IMPL_NESTED(typeName, type) struct WTFTzoneMallocSemicolonifier##typeName { }
+#define WTF_MAKE_TZONE_ALLOCATED_IMPL_NESTED(outerType, innerType) \
+    struct WTFTzoneMallocSemicolonifier ## outerType ## innerType { }
```

Comment:
Not sure we can do that because these are used syntactically as if declaring a global variable i.e. it is followed by a semicolon.  We need to declare those struct placeholders to satisfy the make the use of the semicolon legal.  And we need the arguments to make each structure unique.